pkg/sparrow/run.go

```diff
@@ -62,6 +74,13 @@ func New(cfg *config.Config) *Sparrow {
 		cCfgChecks:  make(chan map[string]any, 1),
 		routingTree: api.NewRoutingTree(),
 		router:      chi.NewRouter(),
+		targets: targets.NewGitlabManager(
+			gitlab.New(gitlabBaseUrl, os.Getenv("GITLAB_TOKEN"), gitlabRegistrationProjectID),
+			"cool-sparrow.de",
```

Comment:
Is this hardcoded value wanted here for now?

If yes could you add a comment, to clarify that this needs to be changed in a future PR?